client/lib/products-values/get-product-billing-term-label.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { TERM_MONTHLY, TERM_ANNUALLY, TERM_BIENNIALLY } from 'calypso/lib/plans/constants';
+import { PRODUCTS_LIST } from 'calypso/lib/products-values/products-list';
+
+/**
+ * Type dependencies
+ */
+import type { ProductSlug } from 'calypso/lib/products-values/types';
+import type { Product } from 'calypso/lib/products-values/products-list';
+
+/**
+ * Returns the billing term label for a product (i.e. "every month", "every year", "every two years").
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'renews every two years', { context: 'Informative text for renewals in /checkout'} )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/purchase-item-status-term)